docs/src/pages/[platform]/index.page.tsx

```diff
@@ -1,333 +1,71 @@
-  Button,
-  Card,
-  Flex,
-  Grid,
-  Heading,
-  Image,
-  Link,
-  Text,
-  ToggleButton,
-  ToggleButtonGroup,
-  View,
-  useTheme,
-  useBreakpointValue,
-} from '@aws-amplify/ui-react';
-import { MdChevronRight } from 'react-icons/md';
 import { Footer } from '@/components/Layout/Footer';
-import { HomeLogo } from '../HomeLogo';
-import { HomePrimitivePreview } from '../HomePrimitivePreview';
-import { theme } from '../../theme';
-import { ThemeButton } from '../ThemeButton';
 import { useCustomRouter } from '@/components/useCustomRouter';
-import { FRAMEWORKS } from '@/data/frameworks';
-import { TerminalCommand } from '@/components/InstallScripts';
-
-// react-live does not work with SSR so we have to load
-// it dynamically and only in the client
-const HomeEditor = dynamic(() => import('../HomeEditor'), { ssr: false });
-
-const AmpCard = ({ title, description, href }) => (
-  <Link isExternal href={href} className="docs-home-amp-product-card">
-    <Card textAlign="center" variation="outlined">
-      <Heading level={3}>{title}</Heading>
-      <Text>{description}</Text>
-    </Card>
-  </Link>
-);
+import { HeroSection } from '@/components/home/sections';
+import ReactHomePage from './index.react';
+import VueHomePage from './index.vue';
+import AngularHomePage from './index.angular';
+import FlutterHomePage from './index.flutter';
 
 const HomePage = ({ colorMode }) => {
   const {
     query: { platform = 'react' },
   } = useCustomRouter();
-  const { tokens } = useTheme();
-  const [themeOverride, setThemeOverride] = React.useState('');
-  const showEditor = useBreakpointValue({
-    base: false,
-    medium: true,
-  });
-
-  const installScripts = {
-    react: `npm i @aws-amplify/ui-${platform} aws-amplify`,
-    vue: `npm i @aws-amplify/ui-${platform} aws-amplify`,
-    angular: `npm i @aws-amplify/ui-${platform} aws-amplify`,
-    flutter: 'flutter pub add amplify_authenticator',
-  };
-  const frameworkInstallScript = installScripts[platform.toString()];
-  return (
-    <View data-amplify-theme-override={themeOverride}>
-      <ThemeProvider theme={theme} colorMode={colorMode}>
-        <View as="section" className="container">
-          <h1 className="docs-home-logo">
-            <HomeLogo />
-          </h1>
 
-          <Flex
-            direction={{ base: 'column', medium: 'row' }}
-            padding={tokens.space.large}
-          >
-            <Card variation="outlined" flex="1">
-              <Text
-                fontSize={{
-                  base: tokens.fontSizes.large,
-                  small: tokens.fontSizes.xl,
-                }}
-              >
-                Amplify UI is an open-source design system with cloud-connected
-                components and primitives that simplify building accessible,
-                performant, and beautiful applications in React, Angular, Vue,
-                and Flutter (more coming soon).
-              </Text>
+  const handleScroll = debounce((e) => {
```

Comment:
What is this used for?

Reply:
this lil effect. Is there a more simpler/efficient way to do this?

![CleanShot 2022-06-21 at 17 27 49](https://user-images.githubusercontent.com/321279/174918216-5a7218a2-9100-4ff7-9b2b-9a270ff2a4c3.gif)